### ISSUE TITLE
[BUG] There is no activation layer in FC !

### ISSUE THREAD

**Issue by illian01** [Nota-NetsPresso/netspresso-trainer#320]
### Describe the bug

Activation layer must be added after intermediate `nn.Linear`.

https://github.com/Nota-NetsPresso/netspresso-trainer/blob/4bb5685117dd76eb2e44ee3088a64d40b4e8a4dd/src/netspresso_trainer/models/heads/classification/core/fc.py#L22-L26

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

N/A

### System Info

```shell
Current dev branch
```

**Comment by illian01** [comment]
Also planned to add `dropout` for last classifier layer.
